src/index.js

```diff
@@ -0,0 +1,58 @@
+'use strict'
+
+const io = require('@pm2/io')
+const CPUMetrics = require('./metrics/cpu.js')
+const NetworkMetrics = require('./metrics/network.js')
+const DiskMetrics = require('./metrics/disk.js')
+const MemoryMetrics = require('./metrics/memory.js')
+const FSMetrics = require('./metrics/fs.js')
+const TTYMetrics = require('./metrics/tty.js')
+const ProcessesMetrics = require('./metrics/processes.js')
+
+const MonitoringActions = require('./actions.js')
+
+io.initModule({
+  widget: {
+    type: 'generic',
+    logo: 'https://www.glcomp.com/media/catalog/category/Dell-R620_3_1_1.png',
+
+    theme: ['#111111', '#1B2228', '#807C7C', '#807C7C'],
+
+    el: {
+      probes: true,
+      actions: true
+    },
+
+    block: {
+      actions: false,
+      issues: true,
+      meta: true,
+      cpu: false,
+      mem: false,
+      main_probes: [
+        'CPU usage',
+        'Free memory',
+        'Avail. Disk',
```

Comment:
Display even if it's not available on current platform?

Reply:
it's only used for keymetrics so it's not really important for me